test/e2e/tests/ppom/migrate-opensea-to-blockaid-banner.spec.js

```diff
@@ -64,7 +64,6 @@ describe('Migrate Opensea to Blockaid Banner @no-mmi', function () {
         title: this.test.fullTitle(),
       },
       async ({ driver }) => {
```

Comment:
There is another case in the file that needs to be removed as well. See above on line 39